### ISSUE TITLE
Thermal Shutdown Response?

### ISSUE THREAD

**Issue by trbLeeciN** [gnea/grbl#753]
Hi there,
I searched all issues, I couldn't find or I missed my problem.
Are there any response message or alarm message if stepper driver got thermal shutdown?
Thanks.

**Comment by bdurbrow** [comment]
No.... there's usually no input back to the processor that Grbl is running on that the steppers have a problem.

If you are having thermal shutdown issues; you really need to sort that out at the hardware level (fan? heatsink? liquid cooling, the way high-performance PC are setup?).
